### PR TITLE
add bls fallback option

### PR DESCRIPTION
**Pull request by ChihChengLiang**
### What was wrong?

blspy is hard to install

### How was it fixed?

opt out blspy if you want.

#### Cute Animal Picture

🦋

### CODE REVIEW

**Comment by cburgdorf** [comment]
:+1: on that approach

File: eth2/_utils/bls/backends/__init__.py [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
     PyECCBackend,
 )
+
+
+# If blspy not installed, use PyECC as default BLS backend
+
+try:
+    from .chia import ChiaBackend
+    DEFAULT_BACKEND = ChiaBackend
+    AVAILABLE_BACKENDS += ChiaBackend
```
Comment (mhchia) [inline eth2/_utils/bls/backends/__init__.py]: Does this line work? It looks like it should fail, but I didn't see errors in the CI(except for the linter).

Comment (cburgdorf) [reply eth2/_utils/bls/backends/__init__.py]: It does work. It returns a new tuple which is then assigned to the old variable. So it is essentially the same as `AVAILABLE_BACKENDS = AVAILABLE_BACKENDS + ChiaBackend`

**Comment by ChihChengLiang** [comment]
@cburgdorf @pipermerriam  The PR is ready to merge if no further objections.

**Comment by pipermerriam** [comment]
No objections, just voicing what I think is the proper *long term* plan.  Short term this is a fine solution.

**Pull request merged**